src/freetype.rs

```diff
@@ -646,7 +646,7 @@ pub const FT_Err_Corrupted_Font_Header: u32 = 185_u32;
 pub const FT_Err_Corrupted_Font_Glyphs: u32 = 186_u32;
 pub const FT_Err_Max: u32 = 187_u32;
 
-#[link(name="freetype")]
+#[link(name="freetype", kind="static")]
```

Comment:
You should not need this line, since servo-freetype-sys has:
https://github.com/servo/libfreetype2/blob/master/build.rs#L26

The one thing that you might need, though, is an `extern crate servo-freetype-sys;` to ensure that Cargo/Rustc do not drop the linkage. The static lib should be embedded into the servo-freetype-sys rlib automatically, and as long as it does not get dropped by the compiler, we'll get it via that route.